src/AsyncTCP.cpp

```diff
@@ -448,8 +448,23 @@ static int8_t _tcp_sent(void *arg, struct tcp_pcb *pcb, uint16_t len) {
   return ERR_OK;
 }
 
-static void _tcp_error(void *arg, int8_t err) {
+void AsyncClient::_tcp_error(void *arg, int8_t err) {
```

Comment:
what is the reason behind making it a class member?

Reply:
to access private fields.